src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -658,38 +659,14 @@ class AddItemsPage extends Component {
 
     if (_.some(itemsMap, item => item.length > 1) && !(this.state.values.origin.type === 'SUPPLIER' || !this.state.values.hasManageInventory)) {
       this.confirmTransition(
-        () => this.saveAndTransitionToNextStep(formValues, lineItems),
+        () => this.transitionToNextStep(),
         _.reduce(itemsWithSameCode, (a, b) => a.concat(b), []),
       );
     } else {
-      this.saveAndTransitionToNextStep(formValues, lineItems);
+      this.transitionToNextStep();
     }
   }
 
-  /**
-   * Saves current stock movement progress (line items) and goes to the next stock movement step.
-   * @param {object} formValues
-   * @param {object} lineItems
-   * @public
-   */
-  saveAndTransitionToNextStep(formValues, lineItems) {
```

Comment:
I am not sure if this is a good idea in terms of "last line dilemma", that we discussed on slack. Please take a look at that case.

Reply:
I solved this by adding debounced save. I removed `saveAndTransitionToNextStep`, because we don't want to performing save if all of our lines are autosaved. Additionally we have `transitionToNextStep` which has the same responsibility as this deleted function, but without performing save.